### PR TITLE
Login & Sign-up

### PR DESCRIPTION
**Pull request by ark2002**
- Added Login Page
- Added Sign-up Page
- Linked all the new pages
- Login Page : https://deploy-preview-5--astromart-screens.netlify.app/loginpage/loginpage.html
- Sign-up Page : https://deploy-preview-5--astromart-screens.netlify.app/signuppage/signuppage.html

### CODE REVIEW

File: loginpage/loginpage.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    height: 90vh;
+
+}
+
+.login__form {
+    gap: 1rem;
+    width: 30%;
+    margin: 10rem auto;
+    padding: 2rem 5rem;
+    box-shadow: #0000003d 0px 3px 8px;
```
Comment (arjunbharti) [inline loginpage/loginpage.css:12]: You can use color variable here

File: signuppage/signuppage.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    padding-bottom: 5rem;
+
+}
+
+.signup__form {
+    gap: 1rem;
+    width: 30%;
+    margin: 10rem auto;
+    padding: 2rem 5rem;
+    box-shadow: #0000003d 0px 3px 8px;
```
Comment (arjunbharti) [inline signuppage/signuppage.css:13]: color variable here too

File: signuppage/signuppage.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  <link rel="stylesheet" href="/common.css" />
+  <link rel="stylesheet" href="https://astro-ui.netlify.app/main.css">
+  <link rel="stylesheet" href="/signuppage/signuppage.css">
+  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css" />
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+</head>
+
+<body>
+  <div class="signuppage__container flex--column">
```
Comment (arjunbharti) [inline signuppage/signuppage.html:19]: You can add underscore between signuppage for use camelCase for better code readability.

**Review by arjunbharti — APPROVED** [review]
Added few suggestions, everything else is fine!

File: loginpage/loginpage.css (hunk 1 above)
Comment (ark2002) [reply loginpage/loginpage.css:12]: this color is just being used for box shadow

File: signuppage/signuppage.html (hunk 3 above)
Comment (ark2002) [reply signuppage/signuppage.html:19]: would remember it forward

File: signuppage/signuppage.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Sign-up page</title>
```
Comment (mohitdhatrak) [inline signuppage/signuppage.html]: No need to mention page word i guess in title, kind of redundant

**Review by mohitdhatrak — COMMENTED** [review]
Good to merge! One imp suggestion -  for yellow buttons, there is a big contrast issue, change text to black

File: signuppage/signuppage.html (hunk 4 above)
Comment (ark2002) [reply signuppage/signuppage.html]: fixed

**Pull request merged**